src/tuplet.ts

```diff
@@ -150,15 +140,15 @@ export class Tuplet extends Element {
    * Set whether or not the bracket is drawn.
    */
   setBracketed(bracketed: boolean): this {
-    this.bracketed = !!bracketed;
+    this.options.bracketed = !!bracketed;
     return this;
   }
 
   /**
    * Set whether or not the ratio is shown.
    */
   setRatioed(ratioed: boolean): this {
-    this.ratioed = !!ratioed;
+    this.options.ratioed = !!ratioed;
```

Comment:
Same here. I'm okay with removing the !! hack to force the parameter to be a boolean.